tools/docker_build_dist.sh

```diff
@@ -15,7 +15,7 @@ done
 
 # set the lalsuite checkout to use
 
-if [ "x$SOURCE_TAG" == "x" ] ; then
+if [ "x$SOURCE_TAG" != "x" ] ; then
```

Comment:
Should this change be merged? Doesn't this mean 'if `$SOURCE_TAG` is **not** empty; set `SOURCE_TAG="master"`'?

Reply:
Yep, I edited in the wrong place ... .Thanks for catching!